### ISSUE TITLE
🐞 sdk(elixir): Running basic example fails

### ISSUE THREAD

**Issue by gmile** [dagger/dagger#5368]
### What is the issue?

Example from the [readme](https://github.com/dagger/dagger/tree/main/sdk/elixir#running) fails with what appears to be a generic issue:

```elixir
Mix.install([{:dagger, github: "dagger/dagger", sparse: "sdk/elixir"}])

client = Dagger.connect!()

client
|> Dagger.Query.container([])
|> Dagger.Container.from(address: "hexpm/elixir:1.14.4-erlang-25.3-debian-buster-20230227-slim")
|> Dagger.Container.with_exec(args: ["elixir", "--version"])
|> Dagger.Container.stdout()
|> IO.puts()
```

### Log output

The output when running the above code is:

```
** (Protocol.UndefinedError) protocol Jason.Encoder not implemented for {:args, ["elixir", "--version"]} of type Tuple, Jason.Encoder protocol must always be explicitly implemented. This protocol is implemented for the following type(s): Any, Atom, BitString, Date, DateTime, Decimal, Float, Integer, Jason.Fragment, Jason.OrderedObject, List, Map, NaiveDateTime, Time
    (jason 1.4.0) lib/jason.ex:164: Jason.encode!/2
    (dagger 0.0.0) lib/dagger/query_builder.ex:47: anonymous fn/1 in Dagger.QueryBuilder.Selection.build_args/1
    (elixir 1.15.0) lib/enum.ex:1794: anonymous fn/2 in Enum.map_join/3
    (elixir 1.15.0) lib/enum.ex:1763: anonymous fn/4 in Enum.map_intersperse/3
    (stdlib 5.0.1) maps.erl:416: :maps.fold_1/4
    (elixir 1.15.0) lib/enum.ex:2522: Enum.map_intersperse/3
    (elixir 1.15.0) lib/enum.ex:1794: Enum.map_join/3
    (dagger 0.0.0) lib/dagger/query_builder.ex:48: Dagger.QueryBuilder.Selection.build_args/1
```

### Steps to reproduce

Run the code excerpt above, see the issue.

Trivia: I'm using Elixir 1.15 and OTP 26.0.1.

### SDK version

Elixir SDK 1.0

### OS version

macOS 13.3 (22E252)

**Comment by gmile** [comment]
cc @wingyplus

**Comment by wingyplus** [comment]
@gmile The `:address` in `Dagger.Container.from` and `:args` in `Dagger.Container.with_exec` should be omit. So the final version should be: 


```elixir
Mix.install([{:dagger, github: "dagger/dagger", sparse: "sdk/elixir"}])

client = Dagger.connect!()

client
|> Dagger.Query.container([])
|> Dagger.Container.from("hexpm/elixir:1.14.4-erlang-25.3-debian-buster-20230227-slim")
|> Dagger.Container.with_exec(["elixir", "--version"])
|> Dagger.Container.stdout()
|> IO.puts()
```

Please keep in  mind that the dagger in repo is now different and breaking change from `dagger_ex` since we need to keep API compatible with another SDK.

And since it doesn't publish to hex doc yet. You can see the API documentation by run the following commands:

```
$ git clone https://github.com/dagger/dagger.git
$ cd dagger # you can ignore this line and git clone above if you're already cloned.
$ cd sdk/elixir
$ mix deps.get
$ mix docs
$ open doc/index.html 
```

**Comment by wingyplus** [comment]
Please help me confirm that it works for you. I'll close the issue after then.

**Comment by gmile** [comment]
Hi, @wingyplus ! Yes, this makes sense, thank you. I figured I needed to change the code shortly after posting this issue, but forgot to report back.

On that basis I think this issue can be closed 👍

One thing I am a bit unsure though, is that the code excerpt above was taken from the Readme.md to the Elixir SDK in current repository and not from former dagger_ex repo. I assumed things in this repo, including Readme.md relate to the lasts dagger. Shouldn't the example in readme be fixed?
